docs/components/connectors/react-components/_connectors-table.css

```diff
@@ -0,0 +1,285 @@
+  display: block;
+  border: 1px solid #d9d9d9;
+  border-radius: 4px;
+  padding: 24px;
+  text-align: left;
+  background-color: #fff;
+  transition:
+    transform 0.3s,
+    box-shadow 0.3s;
+  text-decoration: none;
+  color: inherit;
+}
+
+.connector-card img {
+  height: 40px;
+  border: none;
+  margin-bottom: 10px;
+  margin-top: 0;
+  padding: 0;
+}
+
+.connector-card h3 {
+  margin: 0;
+  font-size: 1.1em;
+  font-weight: 600;
+  padding-bottom: 10px;
+}
+
+.connector-card p {
+  margin: 0;
+  font-size: 0.9em;
+  color: #555;
+}
+
+.connector-card:hover {
+  transform: translateY(-5px);
+  box-shadow: 0 4px 8px rgba(0, 0, 0, 0.2);
+  transition: all 0.3s ease;
+}
+
+.connector-small-grid {
+  display: grid;
+  grid-template-columns: repeat(3, 1fr);
+  gap: 20px;
+  margin-top: 25px;
+  margin-bottom: 30px;
+}
+
+.connector-small {
+  display: flex;
+  align-items: center;
+  padding: 10px;
+  border: 1px solid #d9d9d9;
+  border-radius: 4px;
+
+  margin-bottom: 4px;
+}
+.connector-small img {
+  height: 32px;
+  border: none;
+  padding: 0;
+  margin: 0;
+  padding-right: 10px;
+}
+
+.connector-small:hover {
+  transform: translateY(-1px);
+  box-shadow: 0 4px 8px rgba(0, 0, 0, 0.1);
+}
+
+a.connector-small-link {
+  text-decoration: none;
+  color: inherit;
+}
+
+@media (max-width: 1200px) {
+  .connector-small-grid {
+    grid-template-columns: repeat(3, 1fr);
+  }
+}
+
+@media (max-width: 1024px) {
+  .connector-grid {
+    grid-template-columns: repeat(2, 1fr);
+  }
+  .connector-small-grid {
+    grid-template-columns: repeat(3, 1fr);
+  }
+}
+
+@media (max-width: 768px) {
+  .connector-grid {
+    grid-template-columns: 1fr;
+  }
+  .connector-small-grid {
+    grid-template-columns: 1fr;
+  }
+}
+
+/* double column div layout  */
+.banner-column-container {
+  display: flex;
+  border-radius: 4px;
+  margin-top: 30px;
+  margin-bottom: 30px;
+  background-color: #000;
+}
+.banner-column-left {
+  flex: 1;
+  padding: 40px 10px 40px 30px;
+}
+.banner-column-right {
+  flex: 3;
+  padding: 40px 30px 40px 0px;
+  color: white;
+}
+.banner-column-left img {
+  border: none;
+  padding: 0;
+  margin: 0;
+  width: 160px;
+}
+@media (max-width: 768px) {
+  .banner-column-container {
+    flex-direction: column;
+  }
+  .banner-column-right {
+    padding: 0px 30px 40px 30px;
+  }
+  .banner-column-left {
+    padding: 40px 30px 20px 30px;
+  }
+}
+
+.fade-in-top-image {
+  opacity: 0;
+  transform: translateY(-20px);
+  animation: fadeInTop 2s forwards;
+}
+
+@keyframes fadeInTop {
+  to {
+    opacity: 1;
+    transform: translateY(0);
+  }
+}
+
+.filter-badges {
+  display: inline-block;
+  margin-right: 5px;
+}
+
+.badge--default {
+  display: inline-block;
+  padding: 6px 12px;
+  border-radius: 4px;
+  background-color: #eee;
+  cursor: pointer;
+  margin-right: 10px;
+  font-size: 85%;
+  font-weight: var(--ifm-font-weight-bold);
+  line-height: 1;
+  color: #777;
+}
+
+.badge--default:hover {
+  background-color: #ccc;
+}
```

Comment:
I forgot to mention in my review that the contrast doesn't meet accessibility recommendations on the `default` and `hover` states of these buttons (the two gray buttons in this screenshot):

<img width="253" alt="image" src="https://github.com/user-attachments/assets/53acba00-8126-4e87-95fa-f43e67c513ec" />

My browser tools recommended `#3a3a3a` as the lightest shade of gray that contrasts with both backgrounds enough to meet AAA/optimal compliance.